tests/integration/web/prefixviewset_test.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+import json
+import pytest
+
+from nav.models.manage import Prefix, Vlan, NetType
+from nav.compatibility import force_str
+from nav.web.api.v1.views import get_endpoints
+
+ENDPOINTS = {name: force_str(url) for name, url in get_endpoints().items()}
+prefix_url = ENDPOINTS['prefix']
+
+
+def test_contains_address_filter_returns_prefix_containing_given_address(
+    client, prefix
+):
+    response = client.get(prefix_url, {"contains_address": "10.1.1.0/24"})
+    assert response.status_code == 200
+    content = json.loads(response.content.decode('utf-8'))
+    prefix_ids = [prefix['id'] for prefix in content['results']]
+    assert prefix.id in prefix_ids
+
+
+def test_contains_address_filter_does_not_return_prefix_that_does_not_contain_given_address(
+    client, prefix
+):
+    response = client.get(prefix_url, {"contains_address": "20.4.111.0/24"})
+    assert response.status_code == 200
+    content = json.loads(response.content.decode('utf-8'))
+    prefix_ids = [prefix['id'] for prefix in content['results']]
+    assert prefix.id not in prefix_ids
+
+
+def test_contains_address_filter_fails_if_given_address_is_not_valid_cidr_address(
+    client, prefix
+):
+    response = client.get(prefix_url, {"contains_address": "invalid_address"})
+    assert response.status_code == 400
+
+
+def test_contains_address_filter_returns_prefix_identical_to_given_address(
+    client, prefix
+):
+    response = client.get(prefix_url, {"contains_address": prefix.net_address})
+    assert response.status_code == 200
+    content = json.loads(response.content.decode('utf-8'))
+    prefix_ids = [prefix['id'] for prefix in content['results']]
+    assert prefix.id in prefix_ids
+
```

Comment:
I have one overarching suggestion here:

In order to avoid the tedious repetition of `contains_address_filter` in every test name, the tests can be grouped in a class called `TestContainsAddressFilter`. This signals that all the tests therein are testing behaviors of the same functionality - and this name will be included in the test reports.

Reply:
done